ui/docker.sh

```diff
@@ -29,5 +29,19 @@ DATE="$(date --iso-8601=seconds)"
 LABEL_DATE="--label \"org.opencontainers.image.created=${DATE}\""
 GIT_REV="$(git rev-parse HEAD)"
 LABEL_GIT_REV="--label \"org.opencontainers.image.revision=${GIT_REV}\""
-
+USE_PODMAN=0
+USE_DOCKER=0
+OPTIONS='pd'
+while getopts $OPTIONS OPTION
+do
+        case "$OPTION" in
+                p)USE_PODMAN=1;;
+                d)USE_DOCKER=1;;
+                *)echo "Please choose between -p for podman or -d for docker.";;
```

Comment:
@j-porsche `exit` at  line 40 ?

Reply:
@weizhouapache Yeah. Good point. I'll add this